cmd/archeio/internal/e2e/e2e_containerd_linux_test.go

```diff
@@ -0,0 +1,107 @@
+//go:build linux && !noe2e
+// +build linux,!noe2e
+
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+	"time"
+)
+
+func TestE2EContainerdPull(t *testing.T) {
+	t.Parallel()
+	containerdVersions := []string{"1.6.20", "1.7.0"}
```

Comment:
TODO: keeping these up to date ...

With crane we're floating at the latest tagged release

We can maybe figure out something reasonable to detect the most recent patch release on the current and N-1 minor versions ...